.github/workflows/promote-release.yml

```diff
@@ -58,18 +58,17 @@ jobs:
   change-management:
     needs: [ promote ]
     if: fromJSON(inputs.isStableRelease)
-    runs-on: ubuntu-latest
-    environment: ChangeManagement
-    env:
-      TPS_API_APP_ID: ${{ secrets.TPS_API_APP_ID }}
-      TPS_API_RELEASE_ACTOR_EMAIL: ${{ secrets.TPS_API_RELEASE_ACTOR_EMAIL }}
-      TPS_API_STAGE: ${{ secrets.TPS_API_STAGE }}
-      TPS_API_TOKEN_PARAM: ${{ secrets.TPS_API_TOKEN_PARAM }}
-      TPS_API_URL_PARAM: ${{ secrets.TPS_API_URL_PARAM }}
+    # Failure to record the release should not fail the workflow
+    continue-on-error: true
     steps:
+      # Checkout required to get github.sha
       - uses: actions/checkout@v3
-      - run: yarn --immutable --network-timeout 1000000
-      - run: ./scripts/postrelease/change_management
+      - run: ./scripts/postrelease/tps_record_release cli ${{ github.sha }}
+    environment: ChangeManagement
+    env:
+      ACTOR_EMAIL: ${{ secrets.TPS_API_RELEASE_ACTOR_EMAIL }}
+      STAGE: ${{ secrets.TPS_API_STAGE }}
```

Comment:
it depends on what we mean by stage exactly. we do release to different _channel_s -- stable (which is what customers see by default), alpha, and beta. It's kind of a different concept though, so I don't know if we want to communicate that here.

Reply:
I wonder about these `TPS_API_*` secret values… I don't know what is actually stored in there!

Since these job steps are are being limiting to run only for stable releases, it does seem this could pass `production` value directly.

Reply:
`STAGE` has been removed as an arg, and now always sets `production`.